group.lisp

```diff
@@ -101,6 +101,14 @@ needs to redraw anything on it, this is where it should do it."))
   (:documentation "When a head or its usable area is resized, this is
 called. When the modeline size changes, this is called."))
 
+(defmethod group-current-window (group)
+  (screen-focus (group-screen group)))
```

Comment:
This is causing an issue. The "current window" in a group can now be a window that's not actually in that group, which goes against expectations in other parts of the codebase. (Currently I'm looking at `group-forward-with-window`)